### ISSUE TITLE
[R Kernel] Periodically execute cell OR refresh cell output

### ISSUE THREAD

**Issue by marcozmonteiro** [jupyter/notebook#1446]
Hello,

I'm working on a project that requires a Jupyter R Notebook cell to be executed periodically, refreshing its output (a plot made on ggplot). What's the best way to overcome this problem in Jupyter?

I tried creating an infinite loop in the R code, however, I wasn't able to find a way to clear the output cell, thus, the output cell keeps accumulating many plots (instead of just showing one).

Thank you,

Marcos

**Comment by takluyver** [comment]
You can use nbconvert to run a notebook non-interactively (with the `--execute` flag). You can automate that to run regularly using cron (on Unix systems), or Windows 'scheduled tasks' (does Windows still have that? I haven't used recent versions much). There are browser extensions you can get that automate reloading a page every few minutes. It's not a very integrated solution, but all the pieces are there to do it quite easily.

A bit more challenging would be to write a JS extension that executes a given cell/cells on a schedule.

**Comment by marcozmonteiro** [comment]
Thank you for your answer. I ended up finding this --execute option a while after I created this issue.
I am going towards your solution as well, I'm going to execute the R notebook I have by calling Jupyter inside a Python application to generate the HTML report I want, and also I'm going to use an Add-On for Firefox that auto-reloads a HTML page whenever it is changed.

I appreciate your time and effort, have a nice day.
